web_app/hazard-reporting-map/pages/incidents/index.tsx

```diff
@@ -0,0 +1,371 @@
+                    <DropdownMenuLabel>Type of Aid</DropdownMenuLabel>
+                    <DropdownMenuSeparator />
+
+                    {typeOfAidArr.map((itm) => (
+                      <DropdownMenuItem
+                        key={itm}
+                        onClick={() => handleTypeOfAid(itm)}
+                      >
+                        {itm}
+                      </DropdownMenuItem>
+                    ))}
+                  </DropdownMenuContent>
+                </DropdownMenu>
+
+                <DropdownMenu>
+                  <DropdownMenuTrigger>
+                    {' '}
+                    <Button
+                      className="w-[120px] gap-2 px-6 text-xs capitalize"
+                      variant={'ghost'}
+                      size={'sm'}
+                    >
+                      {district.length > 0 ? district : 'Feni'}{' '}
+                      <ChevronDown size={10} />
+                    </Button>
+                  </DropdownMenuTrigger>
+                  <DropdownMenuContent>
+                    <DropdownMenuLabel>District</DropdownMenuLabel>
+                    <DropdownMenuSeparator />
+
+                    {districtsArr.map((itm) => (
+                      <DropdownMenuItem
+                        key={itm}
+                        onClick={() => handleDistrict(itm)}
+                      >
+                        {itm}
+                      </DropdownMenuItem>
+                    ))}
+                  </DropdownMenuContent>
+                </DropdownMenu>
+
+                <DropdownMenu>
+                  <DropdownMenuTrigger>
+                    {' '}
+                    <Button
+                      className="w-[120px] gap-2 px-6 text-xs capitalize"
+                      variant={'ghost'}
+                      size={'sm'}
+                    >
+                      {thana.length > 0 ? thana : 'Thana'}{' '}
+                      <ChevronDown size={10} />
+                    </Button>
+                  </DropdownMenuTrigger>
+                  <DropdownMenuContent>
+                    <DropdownMenuLabel>Thana</DropdownMenuLabel>
+                    <DropdownMenuSeparator />
+
+                    {thanaArr.map((itm) => (
+                      <DropdownMenuItem
+                        key={itm}
+                        onClick={() => handleThana(itm)}
+                      >
+                        {itm}
+                      </DropdownMenuItem>
+                    ))}
+                  </DropdownMenuContent>
+                </DropdownMenu>
+              </div>
+            </div>
+
+            {/* data table */}
+            <div className="rounded-md">
+              <DataTable columns={columns} loading={false} data={dataSlice} />
+            </div>
+            {/* pagination */}
+            <div className=" flex justify-center md:justify-end px-4 py-6 md:py-2 text-xs items-center mt-8  rounded ">
+              <div className="">
```

Comment:
we can remove the classname for now if it is empty.

Reply:
done